objectrocket/bases.py

```diff
@@ -126,6 +127,52 @@ def __repr__(self):
         )
         return rep
 
+    def acl_sync(self, aws_sync=None, rackspace_sync=None):
+        """Adjust Amazon Web Services and/or Rackspace Acl Sync feature for this instance.
+
+        :param bool aws_sync: True/False whether to enable AWS acl sync for this instance.
+        :param bool rackspace_sync: True/False whether to enable Rackspace acl sync for this instance.
+        """
+        url = self._url + 'acl_sync'
+
+        data = {"aws_acl_sync_enabled": False, "rackspace_acl_sync_enabled": False}
+
+        # Let's get current status of acl sync for this intance to set proper defaults.
+        response = requests.get(url, **self._instances._default_request_kwargs)
+
+        if response.status_code == 200:
+            resp_json = response.json()
+            current_status = resp_json.get('data', {})
+            data.update({"aws_acl_sync_enabled": current_status.get("aws_acl_sync_enabled", False),
+                         "rackspace_acl_sync_enabled": current_status.get("rackspace_acl_sync_enabled", False)})
+            if aws_sync is not None:
+                data.update({"aws_acl_sync_enabled": aws_sync})
+            if rackspace_sync is not None:
+                data.update({"rackspace_acl_sync_enabled": rackspace_sync})
+
+            response = requests.put(url, data=json.dumps(data), **self._instances._default_request_kwargs)
```

Comment:
You can also say `json=data` and the requests lib will JSON encode the dict for you. Convenient, but not critical.